test/validation_test.go

```diff
@@ -56,6 +57,61 @@ func kustomizeBuild(dir string) ([]byte, []byte, error) {
 	return outBuf.Bytes(), errBuf.Bytes(), err
 }
 
+func testNamespaceResources(t *testing.T) {
+	// All namespaces defined in neco-apps should have the `team` label.
+	// Exceptionally, `sandbox` ns should not have the `team` label.
+	targetDirs := map[string]string{
+		"namespaces":      filepath.Join(manifestDir, "namespaces", "base"),
+		"team-management": filepath.Join(manifestDir, "team-management", "base"),
+	}
+
+	type metaData struct {
+		metav1.TypeMeta   `json:",inline"`
+		metav1.ObjectMeta `json:"metadata,omitempty"`
+	}
+
+	for testcase, targetDir := range targetDirs {
+		t.Run(testcase, func(t *testing.T) {
+			stdout, stderr, err := kustomizeBuild(targetDir)
+			if err != nil {
+				t.Error(fmt.Errorf("kustomize build faled. path: %s, stderr: %s, err: %v", targetDir, stderr, err))
+			}
+
+			y := k8sYaml.NewYAMLReader(bufio.NewReader(bytes.NewReader(stdout)))
+			for {
+				data, err := y.Read()
+				if err == io.EOF {
+					break
+				} else if err != nil {
+					t.Error(err)
```

Comment:
```suggestion
					t.Fatal(err)
```

You cannot rely on `data` any more.